code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -117,7 +117,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 					if(H.species.flags & IS_SYNTHETIC)
 						return
 
-				reagents.trans_to_mob(C, REM, CHEM_INGEST, 0.2) // Most of it is not inhaled... balance reasons.
+				reagents.trans_to_mob(C, REM, CHEM_INGEST, 1) // Most of it is not inhaled... balance reasons.

```

Comment:
this is the only part that i'm worried about, but we'll see how it plays out. maybe .5 will be better, but i'm not sure how much ambrosia was nerfed, so as with most balancing things this'll have to be tweaked.